### ISSUE TITLE
Create Contract Not recognizing 1 ETH

### ISSUE THREAD

**Issue by uberdave** [ethereum/mist#1343]
#### System information
``````
Version: `0.8.6`
ubuntu 16.04
Node type:eth loaded with parity

In am trying to deploy the default wallet contract. I have 1ETH in the wallet.
 The create contract screen, will not allow me to post the transaction
``````

[logs.zip](https://github.com/ethereum/mist/files/549545/logs.zip)

![screenshot from 2016-10-24 21-54-55](https://cloud.githubusercontent.com/assets/3323239/19672067/3124e9dc-9a39-11e6-8fbc-bef3e9a36158.png)

![screenshot from 2016-10-24 21-57-56](https://cloud.githubusercontent.com/assets/3323239/19672071/3b046ba8-9a39-11e6-9a6b-2dd75d71d84e.png)

**Comment by uberdave** [comment]
I tried submit;and it seems that the interface is stuck on "confirming". been about an hour now

**Comment by uberdave** [comment]
I am going to try again with ubuntu 14.04. So, I could be a while before I come back. I am closing soon, but I will come back when I have the blockchain downloaded again. I had tried downloading my entire home directory to local, but the db corrupted. Tried the whole thing locally, on ununtu 16.04 previously, but after 3 weeks of failure, I determined that my local machine does not have enough ram. the digital ocean ubuntu 16.04 droplet has 8 gigs of physical ram. It takes about 12 hours, using parity --geth. I have not tried with command line tools. I am going to keep this issue open at ethereum.stackexchange.com . no one there has answered yet. any advice over there, will be appreciated, once this is closed, later today (wed. 10/26/16)
